src/js/tech/tech.js

```diff
@@ -69,32 +70,46 @@ class Tech extends Component {
    * any controls will still keep the user active
    */
   initControlsListeners() {
-    let player = this.player();
-
-    let activateControls = function(){
-      if (player.controls() && !player.usingNativeControls()) {
-        this.addControlsListeners();
-      }
-    };
-
-    // Set up event listeners once the tech is ready and has an element to apply
-    // listeners to
-    this.ready(activateControls);
-    this.on(player, 'controlsenabled', activateControls);
-    this.on(player, 'controlsdisabled', this.removeControlsListeners);
-
     // if we're loading the playback object after it has started loading or playing the
     // video (often with autoplay on) then the loadstart event has already fired and we
     // need to fire it manually because many things rely on it.
     // Long term we might consider how we would do this for other events like 'canplay'
     // that may also have fired.
     this.ready(function(){
+      this.controls_ = !!this.options.controls;
+
+      // Set up event listeners once the tech is ready and has an element to apply
+      // listeners to
+      if (this.controls_ && !this.nativeControls_) {
+        this.addControlsListener();
+      }
+
       if (this.networkState && this.networkState() > 0) {
-        this.player().trigger('loadstart');
+        this.trigger('loadstart');
       }
     });
   }
 
+  setControls(bool) {
+    this.controls_ = bool;
+
+    if (bool) {
+      if (this.controls_ && !this.nativeControls_) {
+        this.addControlsListener();
+      }
+    } else {
+      this.removeControlsListeners();
+    }
+  }
+
+  setNativeControls(bool) {
+    this.nativeControls_ = bool;
+  }
+
+  setUserActive(bool) {
```

Comment:
I think it might be better to leave all the userActive tracking to the player. We could have the tech still trigger click and tap events, and then the player will update it's userActive state when it sees those tech events.